app/components/HomePage/Careers.tsx

```diff
@@ -79,7 +79,7 @@ export default function Careers(props: PageHomeBlocksCareerSection) {
         </Box>
         <SimpleGrid
           spacing={{ base: 12, lg: 4 }}
-          gap={{ base: 19, lg: 200 }}
+          gap={{ base: 19, lg: 90 }}
```

Comment:
This definitely solves the issue with whitespace, but now on 992px <= screen sizes <= 1124px the cards are overlapping. It looks like it could be solved only by decreasing the card size itself. I'm not sure if you have the bandwidth for that right now, so I think it's okay if we leave it like that for now as it happens only for a small range of screen sizes. 
 
![Screen Shot 2024-07-22 at 5 15 17 PM](https://github.com/user-attachments/assets/51cd6f37-7be5-4158-ae37-4d9683b58255)

Reply:
I created a separate ticket for this issue - KM-50